exporter/awsxrayexporter/go.mod

```diff
@@ -2,8 +2,11 @@ module github.com/open-telemetry/opentelemetry-collector-contrib/exporter/awsxra
 
 go 1.14
 
+replace github.com/open-telemetry/opentelemetry-collector-contrib/internal/common => ../../internal/common
+
 require (
-	github.com/aws/aws-sdk-go v1.34.1
+	github.com/aws/aws-sdk-go v1.34.3
+	github.com/open-telemetry/opentelemetry-collector-contrib/internal/common v0.0.0-00010101000000-000000000000
```

Comment:
Hmm - it seems like this should be possible, can you try a bit more, maybe `replace` for the subpackage as well?

Reply:
I actually initially tried to replace the `awsxray` subpackage package instead of the `github.com/open-telemetry/opentelemetry-collector-contrib/internal/common`. Turned out `replace` only works for a Go module with `go.mod`